src/components/ReplayableVideoAsset.vue

```diff
@@ -9,7 +9,18 @@
 -->
 
 <template>
-  <div class="video-replay-container">
+  <div
+    class="video-replay-container"
+    role="group"
+    :aria-roledescription="$t('video.title')"
```

Comment:
I think this `aria-roledescription` attribute will end up getting duplicated in the DOM for one video because of the way that the replayable video component uses the video asset component.

We should probably make sure that there is only one of these present, regardless whether the video is replayable or not, right?

Reply:
This should be fine because `aria-roledescription` is only read by VoiceOver if there is an `aria-label` or `aria-labelledby` in the same element.

In `.video-replay-container` and the `video` elements both have `aria-labelledby` conditionally rendered by `showsDefaultControls` so only one of them will be read by VO